using-valtimo/plugin/zaken-api/configure-plugin.md

```diff
@@ -0,0 +1,47 @@
+# Using the Zaken API Plugin
+
+The Zaken API plugin can be used to store and access data about a Zaak (case) in an application that supports the
+Zaken API standard created and used by the Dutch government.
+
+## Configure the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Zaken API plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the OpenZaak dependency can be found
+[here](../../../valtimo-implementation/modules/documenten-api.md)
```

Comment:
You describe the Zaken API visibility, mention the OpenZaak dependency and link the documenten-api.md